docs/spark-procedures.md

```diff
@@ -421,12 +421,17 @@ Existing data files are added to the Iceberg table's metadata and can be read us
 
 To leave the original table intact while testing, use [`snapshot`](#snapshot) to create new temporary table that shares source data files and schema.
 
+By default, the original table is retained with the name `table_BACKUP_`. You can also explicitly pass  `drop_backup => true` 
```

Comment:
Sorry i should have been more clear, can we remove the second sentence as well?  (as its duplicated already in the flag description).  As you already mention 'by default', I think the user will know to look below.

Reply:
Done, removed the second sentence.